mlrun/pipelines/kfp/v1_8/platform_other.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2024 Iguazio
```

Comment:
how about bundling iguazio.py and platform_other.py under `mounts.py`?

Reply:
Sounds good to me. I'll consolidate these and the remaining mount related functions there.